generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.html.ejs

```diff
@@ -48,7 +48,7 @@
                         <label class="form-control-label" for="newPassword" jhiTranslate="global.form.newpassword.label">New password</label>
                         <input type="password" class="form-control" id="newPassword" name="newPassword"
                                placeholder="{{ 'global.form.newpassword.placeholder' | translate }}"
-                               formControlName="newPassword" #newPassword>
+                               formControlName="newPassword" data-cy="newResetPassword" #newPassword>
```

Comment:
Why not use the same name? I've noticed the same name is used as data-cy value in another file, that's why I ask

Reply:
we definitively need to use the same name